lib/funneler/version.rb

```diff
@@ -0,0 +1,3 @@
+module Funneler
+  VERSION = "1.0"
```

Comment:
ah.. i see what you're saying, my bad!  I think that's inferred, but I'll update that to be certain